src/components/Postiterie.vue

```diff
@@ -206,9 +224,11 @@ export default {
     },*/
 
     sendFile: function(){
-      
-      let fileContent = '<?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" /></root></mxGraphModel>'
-      //let fileContent = this.resultToXML()     // uncomment this line  and remove the one above when resultToXml is ready
+      let $this = this;
+      let fileContent = this.resultToXML()
+      $this.results.forEach(function (res) {
```

Comment:
Don't know what this empty `forEach` is supposed to do.

Reply:
Deleted